rholang/src/main/scala/coop/rchain/rholang/interpreter/Reduce.scala

```diff
@@ -664,22 +665,21 @@ object Reduce {
     }
 
     private[this] def diff: MethodType = { (p: Par, args: Seq[Par]) => (env: Env[Par]) =>
+      def locallyFreeUnion(base: Coeval[BitSet], other: Coeval[BitSet]): Coeval[BitSet] =
```

Comment:
This doesn't appear necessary here.

Reply:
It is, few lines below. I extracted it with the intent to reuse for `EMap`.